pages/posts/[slug].tsx

```diff
@@ -0,0 +1,71 @@
+import matter from "gray-matter";
+
+import BlogPost from "../../landing/blog/BlogPost";
+
+import { Footer, Header } from "../../landing";
+
+import fs from "fs";
+import { join } from "path";
+
+import { getAllPosts, getPostBySlug } from "../../lib/blog";
+
+const postsDirectory = "content/blog/";
+
+interface Props {
+  title: string;
+  authorImage: string;
+  author: string;
+  date: string;
+  content: string;
+}
+
+const BlogPostPage = ({ title, authorImage, author, date, content }: Props) => {
+  return (
+    <>
+      <Header />
+      <BlogPost
+        title={title}
+        authorImage={authorImage}
+        author={author}
+        date={date}
+        content={content}
+      />
+      <Footer bgColor="dark" fgColor="white" />
```

Comment:
```suggestion
      <Footer bgColor="light" fgColor="white" />
```

Probably a light footer would be better here.